Doc/library/venv.rst

```diff
@@ -276,15 +276,15 @@ creation according to their needs, the :class:`EnvBuilder` class.
           the virtual environment.
 
 
-        .. versionchanged:: 3.12
-           The attribute ``lib_path`` was added to the context, and the context
-           object was documented.
-
         .. versionchanged:: 3.11
            The *venv*
            :ref:`sysconfig installation scheme <installation_paths>`
            is used to construct the paths of the created directories.
 
+        .. versionchanged:: 3.12
+           The attribute ``lib_path`` was added to the context, and the context
+           object was documented.
+
```

Comment:
I'm not sure how this found its way into the 3.11 branch, but it shouldn't be here at all! From the perspective of Python 3.11, 3.12 is still the future. Let's delete it while we're here:

```suggestion
```